app/models/concerns/smooch_search.rb

```diff
@@ -100,9 +100,10 @@ def reject_temporary_results(results)
       end
     end
 
-    def parse_search_results_from_alegre(results, after = nil, feed_id = nil, team_ids = nil)
+    def parse_search_results_from_alegre(results, after = nil, feed_id = nil, team_ids = nil, limit = nil)
+      limit ||= CheckConfig.get(:most_relevant_team_limit, 3, :integer)
```

Comment:
Like the other comments: I suggest we always enforce a `limit` to be passed.

Reply:
Done